docs/organizations/roles-and-permissions-for-synced-organizations.md

```diff
@@ -208,7 +208,7 @@ See [managing people](managing-people.md) to list and manage the members of your
 
 ## Configuring who can change analysis configurations {: id="change-analysis-configuration"}
 
-By default, only users with Write permission on a repository can change analysis configurations. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to [ignore issues for your organization](../repositories/issues.md#ignoring-and-managing-issues) and [configure your code patterns](../repositories-configure/code-patterns.md) on your organization.
+By default, only users with Write permission on a repository can change analysis configurations. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to [ignore issues for your organization](../repositories/issues.md#ignoring-and-managing-issues), [ignore files](../repositories-configure/ignoring-files.md) and [configure your code patterns](../repositories-configure/code-patterns.md) on your organization.
```

Comment:
A small suggestion: since we're starting to have lots of operations listed in the same sentence, I think that a shorter link text helps to isolate and identify the separate operations that are supported:

```suggestion
By default, only users with Write permission on a repository can change analysis configurations. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to [ignore issues](../repositories/issues.md#ignoring-and-managing-issues) for your organization, [ignore files](../repositories-configure/ignoring-files.md) and [configure your code patterns](../repositories-configure/code-patterns.md) on your organization.
```